website/docs/r/lb_virtual_server.html.markdown

```diff
@@ -0,0 +1,206 @@
+---
+layout: "vcd"
+page_title: "vCloudDirector: vcd_lb_virtual_server"
+sidebar_current: "docs-vcd-resource-lb-virtual-server"
+description: |-
+  Provides an NSX edge gateway load balancer virtual server resource.
+---
+
+# vcd\_lb\_virtual\_server
+
+Provides a vCloud Director edge gateway load balancer virtual server resource. Adds an edge gateway
+internal or uplink interface as a virtual server. A virtual server has a public IP address and services all incoming client requests. 
+
+~> **Note:** To make load balancing work one must ensure that load balancing is enabled on edge gateway (edge gateway must be advanced).
+This depends on NSX version to work properly. Please refer to [VMware Product Interoperability Matrices](https://www.vmware.com/resources/compatibility/sim/interop_matrix.php#interop&29=&93=) 
+to check supported vCloud director and NSX for vSphere configurations.
+
+~> **Note:** The vCloud Director API for NSX supports a subset of the operations and objects defined in the NSX vSphere 
+API Guide. The API supports NSX 6.2, 6.3, and 6.4.
+
+Supported in provider *v2.4+*
+
+## Example Usage 1 (HTTP virtual server)
+
+```hcl
+provider "vcd" {
+  user     = "${var.admin_user}"
+  password = "${var.admin_password}"
+  org      = "System"
```

Comment:
This should be "sysorg", to avoid clashes with the "org" below. 
Or the provider section could be removed altogether

Reply:
Removed. I actually think it's not worth to set the `provider` everywhere. Noted to cleanup in other Lb examples. I think the index page shows how to use it and it's quite standard accros providers. https://www.terraform.io/docs/providers/vcd/index.html